apps/election-manager/src/App.test.tsx

```diff
@@ -279,7 +278,8 @@ it('tabulating CVRs', async () => {
   // When there are no CVRs imported the full tally report is labeled as the zero report
   fireEvent.click(getByText('View Unofficial Full Election Tally Report'))
   // Verify the zero report generates properly
-  expect(getByTestId('tally-report-contents')).toMatchSnapshot()
+  // TODO: Do this without a snapshot.
+  // expect(getByTestId('tally-report-contents')).toMatchSnapshot()
```

Comment:
maybe we should keep this test until we find an alternative?

Reply:
Alternative found!
```
  // Verify the zero report generates properly
  expect(getAllByText('0').length).toBe(40)
```